### PR TITLE
azure-cli (+deps), gdcm/swig, vtk-osmesa (to archive)

### PR DESCRIPTION
**Pull request by Maxyme**
New branch for new commits. Contains azure package dependencies and GCDM package with pinning of swig.

### CODE REVIEW

File: adal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  version: 0.4.4
+
+source:
+  fn: adal-0.4.4.tar.gz
+  url: https://github.com/AzureAD/azure-activedirectory-library-for-python/releases/download/0.4.4/adal-0.4.4.tar.gz
+
+build:
+  number: 0
+  #noarch_python: True
+  script: python setup.py install
```
Comment (Korijn) [inline adal/meta.yaml]: conda-forge has some good reasons that I can't find a reference for at the moment to use this specific line:

```yaml
  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment (Korijn) [inline adal/meta.yaml]: conda-forge has some good reasons that I can't find a reference for at the moment to use this specific line:

```yaml
  script: python setup.py install --single-version-externally-managed --record record.txt
```

File: azure-cli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: azure-cli
+  version: 0.1.1b1
+
+#source:
+#  fn: azure-cli-0.1.1b1.tar.gz
+#  url: https://pypi.python.org/packages/c0/d9/335f50396dbfcc60e97c17c043a5d96f210c083384c3800885a30d7e5d01/azure-cli-0.1.1b1.tar.gz
```
Comment (Korijn) [inline azure-cli/meta.yaml]: uh this package has no source?

File: azure-cli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+build:
+  # since there are python 2 specific dependencies, we cant use noarch_python
+  noarch_python: True
```
Comment (Korijn) [inline azure-cli/meta.yaml]: why use it then?

File: azure-cli/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+requirements:
+  build:
+    - python
+    - pip
+    - cryptography
+    - cffi
+
+  run:
+    - python
```
Comment (Korijn) [inline azure-cli/meta.yaml]: missing some runtime requirements I think?

File: certifi/meta.yaml [hunk 5]
```diff
@@ -1,25 +1,30 @@
...
-    url: https://pypi.io/packages/source/c/certifi/certifi-2016.9.26.tar.gz
+  fn: certifi-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/certifi/certifi-{{ version }}.tar.gz
+  sha256: 8275aef1bbeaf05c53715bfc5d8569bd1e04ca1e8e69608cc52bcaac2604eb19
 
 build:
-    number: 0
-    script: python setup.py install
+  number: 0
+  script: python setup.py install
```
Comment (Korijn) [inline certifi/meta.yaml]: also needs the specific line

File: cryptography/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  version: 1.7.1
+
+source:
+  fn: 1.7.1.tar.gz
+  url: https://github.com/pyca/cryptography/archive/1.7.1.tar.gz
+
+build:
+  number: 0
+  #noarch_python: True
+  script: python setup.py install
```
Comment (Korijn) [inline cryptography/meta.yaml]: also

File: msrest/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  version: 0.4.4
+
+source:
+  fn: msrest-0.4.4.zip
+  url: https://pypi.python.org/packages/86/6f/be0fdcd61133b9a7b77175c3690b5691170174b8876664063618e3e5847a/msrest-0.4.4.zip
+
+build:
+  number: 0
+  #noarch_python: True
+  script: python setup.py install
```
Comment (Korijn) [inline msrest/meta.yaml]: also

File: cryptography/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
+package:
```
Comment (ivoflipse) [inline cryptography/meta.yaml]: I don't think we should be maintaining our own version of cryptography, we should be copying the version from defaults if we want a copy on our own account

File: azure-cli/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+build:
+  # since there are python 2 specific dependencies, we cant use noarch_python
+  noarch_python: True
+  number: 0
+  script: pip install --user azure-cli
+
+requirements:
+  build:
+    - python
+    - pip
```
Comment (ivoflipse) [inline azure-cli/meta.yaml]: I haven't seen packages require pip before. It's shipped with Python, so I'm not sure what it's necessary for. Perhaps hiding the requirement of setuptools?

File: azure-cli/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: azure-cli
+  version: 0.1.1b1
+
+#source:
+#  fn: azure-cli-0.1.1b1.tar.gz
+#  url: https://pypi.python.org/packages/c0/d9/335f50396dbfcc60e97c17c043a5d96f210c083384c3800885a30d7e5d01/azure-cli-0.1.1b1.tar.gz
+
+build:
+  # since there are python 2 specific dependencies, we cant use noarch_python
```
Comment (ivoflipse) [inline azure-cli/meta.yaml]: We don't have to support Python 2 right?

File: adal/meta.yaml (hunk 1 above)
Comment (ivoflipse) [inline adal/meta.yaml]: Here's the issue talking about it https://github.com/conda-forge/staged-recipes/issues/528

**Comment by Maxyme** [comment]
For review should be only GDCM and Swig the rest is part of a previous workflow that will need to be ironed out in time. Also vtk-osmesa could be in until we have a proper windows version.

**Comment by Korijn** [comment]
I'll go ahead and reject this PR for now since it was superseded by #11

**Pull request closed without merging**